.github/workflows/ci.yml

```diff
@@ -11,28 +11,42 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  build_and_package:
+  windows:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu, windows, macos]
         raytracing_label: [raytracing, no-raytracing]
-        build_type: [standard]
-        include:
-          - egl_label: no-egl
-          - build_type: headless
-            os: ubuntu
-            raytracing_label: raytracing
-            egl_label: egl
 
-    runs-on: ${{matrix.os}}-latest
-    container: ${{ matrix.os == 'ubuntu' && 'ghcr.io/f3d-app/f3d-superbuild-ci' || null }}
+    runs-on: windows-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        path: 'source'
+        submodules: true
+        lfs: true
+
+    - name: Build and package F3D
+      uses: ./source/.github/actions/f3d-superbuild/
+      with:
+        f3d_version: origin/master
+        raytracing_label: ${{matrix.raytracing_label}}
+
+  linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        raytracing_label: [raytracing, no-raytracing]
+        egl_label: [no-egl, egl]
+
+    runs-on: ubuntu-latest
+    container: ghcr.io/f3d-app/f3d-superbuild-ci
 
     env:
       DISPLAY: :0
 
     steps:
-
```

Comment:
I see that you are removing this line all the time, any reason why ?

Reply:
looks weird without new line, but I'll revert the change

Reply:
it does looks weird without it, why removing it ?